static/examples/cstg-secure-signals-example/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</title>
+    <link rel="stylesheet" type="text/css" href="./styles/ads.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/app.css" />
+    <link rel="shortcut icon" href="images/favicon.png" />
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+
+    </script>
+  </head>
+  <body>
+    <h1>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the
+      <a href="https://unifiedid.com/docs/guides/publisher-client-side"
```

Comment:
Use updated URL: https://unifiedid.com/docs/guides/integration-javascript-client-side
(there is a redirect, but we should update to the new URL everywhere we have the old one)

Reply:
is there an EUID version of this?  Or should the EUID docs point to unifiedid?